doc/tech/release_notes/baldy.md

```diff
@@ -15,9 +15,10 @@
 
 | Reference # 	| Feature 	| Notes 	|
 |-------------	|-----------------	|-------	|
-| `baldy.tech.1` 	| VNF Revolution policy and strategy 	|  	|
-| `baldy.tech.2` 	| Future Roadmap |  |
-| `baldy.tech.3` 	| TBD |  |
+| `baldy.tech.1` 	| VNF Evolution policy and strategy 	|  	|
+| `baldy.tech.2` 	| Backwared/Forward Compitability 	|  	|
```

Comment:
nit: Backward

Reply:
Again I disagree with tag diff between RI and RC.
We can continue this way If it pleases everyone else.

Reply:
@CsatariGergely resolved.